src/validationplots/plots.jl

```diff
@@ -18,12 +37,68 @@ end
 """
 # TODO : desc
 """
-function probplot(fm::MaximumLikelihoodEVA)::Plot
+function probplot_data(fm::BayesianEVA)::DataFrame
```

Comment:
`probplot_data` is the only plot data function that does not include a function for the `pwmEVA` structure. I think it should be added.

Reply:
Done